prov/verbs/src/fi_verbs.c

```diff
@@ -943,24 +968,30 @@ static int fi_ibv_getinfo(uint32_t version, const char *node, const char *servic
 {
 	struct rdma_cm_id *id;
 	struct rdma_addrinfo *rai;
-	struct fi_info *fi;
+	struct fi_info *fi, *_info;
```

Comment:
The use of a leading underscore for a user variable name is discouraged.  This is often reserved by the compiler.